python/hail/api2/table.py

```diff
@@ -529,23 +708,129 @@ def drop(self, *exprs):
     def export(self, output, types_file=None, header=True, parallel=False):
         """Export to a TSV file.
 
-        :param output:
-        :param types_file:
-        :param header:
+        Examples
+        --------
+        Export to a tab-separated file:
+
+        >>> table1.export('output/table1.tsv.bgz')
+
+        Note
+        ----
+        It is highly recommended to export large files with a ``.bgz`` extension,
+        which will use a block gzipped compression codec.
```

Comment:
Might suggest: "which can be read natively with Python's `gzip.open` and R's `read.table` natively"